Examples/Genrefy/Genrefy/Model/KeychainService.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  KeychainService.swift
+//  Genrefy
+//
+//  Created on 01/04/2020.
+//  Copyright © 2020 digi.me. All rights reserved.
+//
+
+import Foundation
+import Security
+
+class KeychainService: NSObject {
+    
+    static let shared = KeychainService()
+
+    func updateEntry(data: Data, for key: String) {
+        let keychainQuery = [
+        kSecClass as String       : kSecClassGenericPassword as String,
+        kSecAttrAccount as String : key,
+        kSecValueData as String   : data ] as [String : Any]
+        
+        let status = SecItemUpdate(keychainQuery as CFDictionary, [kSecValueData as String : data] as CFDictionary)
+        
+        if
+            status != errSecSuccess,
+            let err = SecCopyErrorMessageString(status, nil) {
+            print("Keychain wrapper read entry failed: \(err)")
```

Comment:
I would modify the print statement to `"KeychainService: ...` for each occurrence of `print()` statement in this class.